### ISSUE TITLE
4D5308AB - Gears of War 3

### ISSUE THREAD

**Issue by Gh0stblade** [xenia-project/game-compatibility#30]
Tested with: https://github.com/benvanik/xenia/commit/6d4d1db3106028cce6e28b5cb099324e146a7a25

You can hear the first second of video audio playing for the logos then it completely locks up.

Suggested tags: state-nothing

**Comment by Blackbird88** [comment]
Pretty please add Marketplace link
http://marketplace.xbox.com/cs-CZ/Product/Gears-of-War-3/66acd000-77fe-1000-9115-d8024d5308ab

**Comment by Gh0stblade** [comment]
Updated! New suggested tags: gpu-missing-drawing, state-title.

**Comment by tmccaff** [comment]
@Gh0stBlade Can you try the new commit? I think fixed the audio problem and running smoother in games that I know had problems. I just don't have this game.
https://ci.appveyor.com/project/benvanik/xenia/build/1.0.756-master/job/s7x2geexs1w5jtn1/artifacts

**Comment by Sleepy93** [comment]
# Update
The audio problem looks fixed but the processing is a bit slowly.

# Tested 
On (https://github.com/benvanik/xenia/commit/38824bbd733ebbb2e78fe6df772f4a4bcfc6a76c)

# Issues

>!> 00000028 ResolvePath(UPDATE:\) failed - no root found

# Unimplemented
- XFileSectorInformation
- !> 00000028 Unimplemented XLIVEBASE message app=000000FC, msg=00058046, arg1=0021E000, arg2=4018CF40
- XamPartyGetUserList
- XamPartySendGameInvites
- XamPartySetCustomData
- XamPartyGetBandwidth
- XamShowPartyUI
- XamShowCommunitySessionsUI
- XamVoiceSetMicArrayIdleUsers

# Undefined

- !> 00000028 undefined extern call to 82FC819C NetDll_getsockopt
- !> 00000028 undefined extern call to 82FC819C NetDll_getsockopt

# GPU

Writes to unk. register
> - 0081
> - 0082
> - 0E00
> - 0E40

# Log file

[GearsOfWar3.txt](https://github.com/xenia-project/game-compatibility/files/1545898/GearsOfWar3.txt)

# Labels
- state-title
- gpu-missing-drawing (black screen)
- kernel-unimplemented-features

**Comment by SirHayek** [comment]
any progress?

**Comment by Gh0stBlade** [comment]
I doubt it, last time I checked still black screen.

**Comment by BlaZk0** [comment]
any hope that we can play gears 2 and 3 in the near future ?

**Comment by Yahikon** [comment]
Gh0stBlade you have very fail what you done with Soul Reaver alpha version for ps1

**Comment by Gh0stBlade** [comment]
@Yahikon Excuse me?

**Comment by Yahikon** [comment]
Gh0stBlade nope I not forgiveing cause you lost this incredible alpha version

**Comment by Gh0stBlade** [comment]
@Yahikon What on earth are you talking about?

**Comment by Yahikon** [comment]
Gh0stBlade please don't play dumb ,you know what I mean Legacy of Kain Soul Reaver !

**Comment by Gh0stBlade** [comment]
@Yahikon Do not talk to me like this. What on earth is Legacy of Kain Soul Reaver?!?!? Is this a threat?!?!

**Comment by PersonalityPi** [comment]
Can we not spam git issues with pointless arguments. Thank you.

**Comment by gibbed** [comment]
*Any further nonsense comments will result in project bans.*

**Comment by Yahikon** [comment]
I'm sorry but you deserve this,that why i tell you so roud,cause you sold
it to some asshole,and dont tell me what you not understand what i talking
about,all you know PS1 game Soul Reaver Alpha build

31 Май 2018 г. 22:44 пользователь "Gh0stBlade" <notifications@github.com>
написал:

**Comment by Yahikon** [comment]
Gh0stBlade how I can contact with you bout the sale of the alpha version of Soul River for PS1?

**Comment by Yahikon** [comment]
gibbed dear develop!don't touch my comment!you dond understand what I want!

**Comment by Gh0stBlade** [comment]
`JUN 29, 2018`
Tested with: https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299

Black screen, controls don't work but title screen music plays.

**Comment by TGLTClutch** [comment]
could someone try this game again?

**Comment by Margen67** [comment]
@TGLTClutch No difference.

**Comment by ThatAnnoyedGuy** [comment]
Do we know what caused the game to get stuck on a black screen?

Keep up the great work

**Comment by TGLTClutch** [comment]
@ThatAnnoyedGuy Unreal engine has a different way of allocsting memory and xenia doesn't have the required features implemented yet therefore most(if not all) Unreal Engine games get stuck at intro using Xenia.

**Comment by FranciscoCarlosJunior** [comment]
![xenia_2020_04_20_12_12_06_244](https://user-images.githubusercontent.com/60429667/79774503-e09f5b00-8300-11ea-9b56-6da65d8d6027.jpg)
![xenia_2020_04_20_12_12_48_584](https://user-images.githubusercontent.com/60429667/79774509-e301b500-8300-11ea-82cc-22b9ab0417e7.jpg)
![xenia_2020_04_20_12_16_05_891](https://user-images.githubusercontent.com/60429667/79774510-e301b500-8300-11ea-91ad-0b51af9321d8.jpg)
It gets past the intro for me, but then it just goes to a black screen but i can hear the music.
Nothing Changed.

**Comment by FranciscoCarlosJunior** [comment]
log
[xenia.log](https://github.com/xenia-project/game-compatibility/files/4505264/xenia.log)

**Comment by FranciscoCarlosJunior** [comment]
Hey i found solution.😎
Now Playable with XENIA-Custom.😁 
![xenia-custom 1 11g_2020_04_22_11_56_55_969](https://user-images.githubusercontent.com/60429667/80004679-9f3db580-8490-11ea-9ece-29c7a17bbcb3.jpg)
![xenia-custom 1 11g_2020_04_22_11_55_46_595](https://user-images.githubusercontent.com/60429667/80004687-a2d13c80-8490-11ea-91f0-a05878a3f757.jpg)
😎😎

**Comment by AlexByte** [comment]
@FranciscoCarlosJunior you can play with xenia-canary

**Comment by FranciscoCarlosJunior** [comment]
Sure

**Comment by FranciscoCarlosJunior** [comment]
thankyou for you hint

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/c2e8c5554dcb89099952037571fd9d715f982b6f

Played the prologue until a certain somebody died. Xenia died too. 62FPS with vsync off and shaders cached. Probably Playable. 
This is a very gpu demanding game. No more gpu issues.

![got (1)](https://user-images.githubusercontent.com/31154606/94370667-e3c02380-00b6-11eb-9632-fa8fa389637c.png)
![got (3)](https://user-images.githubusercontent.com/31154606/94370702-1538ef00-00b7-11eb-9c98-12f97308531a.png)
![got (4)](https://user-images.githubusercontent.com/31154606/94370704-1ec25700-00b7-11eb-9db6-98bd5512f09d.png)
![got (5)](https://user-images.githubusercontent.com/31154606/94370710-271a9200-00b7-11eb-9337-edc869409c94.png)
![got (6)](https://user-images.githubusercontent.com/31154606/94370716-2da90980-00b7-11eb-9100-af21b47daebc.png)
![got (7)](https://user-images.githubusercontent.com/31154606/94370719-34378100-00b7-11eb-8300-7c2b7e56dd44.png)
![got (8)](https://user-images.githubusercontent.com/31154606/94370725-3568ae00-00b7-11eb-9435-b7b935de27df.png)
![got (9)](https://user-images.githubusercontent.com/31154606/94370727-3699db00-00b7-11eb-9a55-a375af791635.png)
![got (10)](https://user-images.githubusercontent.com/31154606/94370730-38639e80-00b7-11eb-8d6a-58881a74899c.png)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/5288685/xenia.zip)

state-gameplay

**Comment by Silergo** [comment]
Broke script when defeat berserker lambient. Cutscene doesn't start.

**Comment by lokeshsharmaj** [comment]
Seems playable to me I am at Act 2 Chapter 7 without any issue except Pre-rendered BINK videos playing is very slow. 5-10 fps with  extreme stuttering (must be due to some encryption or something to bink videos is applied, didn't happen with Gears 2 which didn't have bink encryption or something). In game videos work fine, game works fine, no issues.

**Comment by lokeshsharmaj** [comment]
> Some spotlights see-through objects.
> ![654654](https://user-images.githubusercontent.com/32046715/106322390-de523000-6286-11eb-9261-30e3a3fa43bb.png)

Yes this particular issue is everywhere. Same issue with Gears Judgement, Gears 2 and 3.

**Comment by lokeshsharmaj** [comment]
there is a bug in Act 3 Chapter Breakneck Run - After the cutscene the pole breaks and kills marcus. No matter what we do it kills us  which never happen in game. In game pole breaks and we pass that without any damage. Tried Both  Xenia Master and Xenia Canary version 04 March 2021.

**Comment by lokeshsharmaj** [comment]
Finished the game.
There are some bugs/Issues
1. Prerendered/Bink Cinematics has issues - Extreme Stuttering as mentioned above.
2. Sometimes game gets stuck at some point and things happen which doesn't happen in game. I faced this situation at two points, FIRST at Act 3: Chapter 3: Breakneck Run where reever in the way but doesn't show and Marcus Die. SECOND Before Entering the Tower where, there are two sides to go to tower, game got stuck for me, there was no one left to kill but it was showing to kill everyone. for both points restarting the chapter and not dying worked (Issue may occur again if we die on our first try, then we need to restart chapter again, at least in SECOND point i had to restart the chapter again if i die once, coz Issue happen again if i die on first try). 
3. There are some graphical bugs, and those are random and dynamic. Restarting the Xenia fixes these graphical bugs.

Game is amazing, but these are some Issues I faced but I enjoyed the game.

**Comment by bomabomaboma** [comment]
> Some spotlights see-through objects.
> ![654654](https://user-images.githubusercontent.com/32046715/106322390-de523000-6286-11eb-9261-30e3a3fa43bb.png)


This issue can be completely fixed by disabling light-shafts in the UE3 .ini files, it doesn't compromise image quality in the least.
You can also modify MSAA level to either clean up the image or improve framerate.

I'm going to try and find addresses that can be patched, because modifying Coalesced_INT.bin with a hex editor without upsetting the TOC is far from ideal.

**Comment by lokeshsharmaj** [comment]
> > Some spotlights see-through objects.
> > ![654654](https://user-images.githubusercontent.com/32046715/106322390-de523000-6286-11eb-9261-30e3a3fa43bb.png)
> 
> This issue can be completely fixed by changing "LensFlares" value to "False" in the UE3 .ini files, and it doesn't compromise image quality in the least.
> You can also modify MSAA level to either clean up the image or improve framerate.
> 
> I'm going to try and find addresses that can be patched, because modifying Coalesced_INT.bin with a hex editor without upsetting the TOC is far from ideal.
> 
> BTW, my experience was near perfect 60fps throughout. I didn't have the stuttering issues listed above.
> Ryzen 9 4900HS, RTX 2060 Max-Q, 16GB RAM
> 
> I wouldn't recommend aspiring for 60fps unless you have an NVIDIA or newer AMD card with at least 5-6 tFLOPS. And I'd imagine 2x SSAA is impossible unless you have a 3080.

There is no such thing as UE3.ini for these games.

**Comment by bomabomaboma** [comment]
No, it's still here. Just not as easy to access as most PC UE3 games.

Coalesced_INT.bin is a version of Unreal Engine 3's .ini files, which use custom headers and layout, which can be viewed in either a custom tool (which I won't mention here), or via editing through Hex editor.

**Comment by Pablo96** [comment]
I just found like a bug I dont know if it is the game or the emulator. Everytime I kill the last spawned lambent in this scene (first mission as cole) the game just suddenly kills me.

![2021-05-21 18_50_30-Window](https://user-images.githubusercontent.com/13873482/119201760-e4eafa00-ba65-11eb-842a-dcb5119b1de5.png)
![2021-05-21 18_51_49-Window](https://user-images.githubusercontent.com/13873482/119201774-ea484480-ba65-11eb-9c06-c7f9fa914c1c.png)
![2021-05-21 18_52_17-Window](https://user-images.githubusercontent.com/13873482/119201778-ed433500-ba65-11eb-86ee-9095dc490e4f.png)

**Comment by bomabomaboma** [comment]
This is a known bug with Gears of War 3 kismet scripting in Xenia
You have three options:
- Restart the chapter and play the entire way through without dying once.
- Import a 100% savegame and start the game beyond the chapter that is bugged.
- Using the Gears of War 2 Editor, edit the Coalesced_Int.bin file to unlock all chapters.

**Comment by aphelion1** [comment]
i don't think that is an option, the game most be playable, if there is a emulation bug it should be fixed, i also have this issue, i tried using the gow 2 editor but is not possible at least because i am using a iso, and the editor only works with .bin, any ideas?

**Comment by RomainGETI** [comment]
Hi there, i'm having trouble playing Gears 3 with Xenia, whenever i launch the game, the intro cutscene with all the Epic Games and Microsoft things goes, and whenever i press start to play, the game just crashes when it has to read the already written saves. 
It's the first time i'm using Xenia, and if there something that I missed, i would be glad if anyone could point it out for me ( sorry for my bad english :) )

**Comment by TheoX75** [comment]
Hi! I am currently running Xenia (Master @211cc99f4 on Nov 12 2021) running Gears of War 3! I am currently in Act IV Bon Voyage on Hardcore, After beating all the Armored Kentas and all other enemies, the game does not continue. I try to turn the wheel move the stuff and Marcus keeps saying "cant do that yet" is this a know bug?

Here is my system specs.
http://speccy.piriform.com/results/VgdfDAodzkA8aPLr5T6OMFp

**Comment by Kyogre** [comment]
every such problems solved by the first solution in [this post](https://github.com/xenia-project/game-compatibility/issues/30#issuecomment-846288392). If one restart didn't help try again and again until the bug won't appear. And yes, it is a known save corruption bug. I faced it too in this armored kantus battle.

**Comment by TheoX75** [comment]
> every such problems solved by the first solution in [this post](https://github.com/xenia-project/game-compatibility/issues/30#issuecomment-846288392). If one restart didn't help try again and again until the bug won't appear. And yes, it is a known save corruption bug. I faced it too in this armored kantus battle.

Any links where I can get a save game past that point or a Save Editor since Google gives dead links?

**Comment by Kyogre** [comment]
> where I can get a save game past that point or a Save Editor 
I never used such things. Cant help you there. I recommend you use first option, not second or third.

**Comment by NexusSomnia** [comment]
Just got Xenia so I'm trying to get the hang of it.
Rendered graphics in Gears 3 come out as a barage of vertical green lines.  Other assets are fine such as menus (first screenshot, the menu itself is fine, the images behind it are not) and loading screens, but gameplay is absolutely unintelligible.
I'm using a GTX 1050Ti, Ryzen 7 2700X, 32GB of RAM
![Screenshot 2021-12-11 202730](https://user-images.githubusercontent.com/95989242/145697096-9dd4f146-bd12-445a-b8b2-1ade37cf215f.jpg)
![Screenshot 2021-12-11 202754](https://user-images.githubusercontent.com/95989242/145697101-a7875fec-4bd4-47cf-8ccf-d699950c814a.jpg)
![Screenshot 2021-12-11 202826](https://user-images.githubusercontent.com/95989242/145697105-f9b31363-e24c-4c25-b83c-4424fd4ed531.jpg)
.

**Comment by Kyogre** [comment]
@NexusSomnia are you using 2K resolution and not using xenia-patches? If so, switch to 720p and apply the patches for GoW3, then test again. If I recall correctly that bug is discussed [here on reddit](https://www.reddit.com/r/xenia/comments/mtd8r0/gears_of_war_3_rtx_2060_bug_green_texture_in_2k/).

**Comment by NexusSomnia** [comment]
> @NexusSomnia are you using 2K resolution and not using xenia-patches? If so, switch **resolution_scale** to **1** and enable [patches](https://github.com/xenia-canary/game-patches/) for GoW3, then test again. If I recall correctly that bug is discussed [here on reddit](https://www.reddit.com/r/xenia/comments/mtd8r0/gears_of_war_3_rtx_2060_bug_green_texture_in_2k/).

Do I need to use Xenia Canary for patches to work?  I'm using Master

**Comment by NexusSomnia** [comment]
> @NexusSomnia are you using 2K resolution and not using xenia-patches? If so, switch **resolution_scale** to **1** and enable [patches](https://github.com/xenia-canary/game-patches/) for GoW3, then test again. If I recall correctly that bug is discussed [here on reddit](https://www.reddit.com/r/xenia/comments/mtd8r0/gears_of_war_3_rtx_2060_bug_green_texture_in_2k/).

Nvm that solved the issue!  Now I've got terrible framerates to grapple with lol but that may be a me issue.

**Comment by dante3732** [comment]
There are places in these game where the game crashes constantly, but it also crashes randomly. Driver give TDR error.
It happened beginning with jun 23 2020 build and still not fixed.
There are no crashes on previous builds.
Windows 11 Pro x64 21H2
AMD Ryzen 5 3600 4.2 Ghz
MSI Radeon VEGA 64 8 Gb HBM2 22.5.1
2x8 Gb DDR4 3200

**Comment by jfish871** [comment]
Played through the game on the latest build of xenia canary as of Sept 5 2022. Faced an issue where reloading from a checkpoint would soft lock the game, reloading from a chapter was the only solution. Also there was an issue where some lens flairs would appear when they should not. Apart from those issues the game is fully playable from start to completion. 
![Incorrect Sorting 1](https://user-images.githubusercontent.com/95584413/190691339-2d0bc61b-4e1f-49e9-8354-8a322175a2a4.PNG)
![Incorrect Sorting 2](https://user-images.githubusercontent.com/95584413/190691340-cf32670b-ecab-4aa4-8b30-8c74ffe8525c.PNG)
![Incorrect Sorting 3](https://user-images.githubusercontent.com/95584413/190691341-241aee7c-ce6c-49ed-90fe-bce9882c661b.PNG)
![Incorrect Sorting 4](https://user-images.githubusercontent.com/95584413/190691343-4bc2b365-5012-4974-abab-0de3de97bf53.PNG)

**Comment by sidhara** [comment]
any solution on the restart chapter problem, the save corruption one? playing the main campaign in insane it's hard but manageable, but the raam shadow campaign it's not viable with that bug

**Comment by m013** [comment]
stuck at act 2, chapter 3 - 'unbreakable'
tried to restart, different versions, canary and master, no success
![Capture](https://github.com/xenia-project/game-compatibility/assets/37451707/e5c08947-b4e8-47d9-8248-608607ce0786)
[xenia.log](https://github.com/xenia-project/game-compatibility/files/12164813/xenia.log)

**Comment by Potato302** [comment]
> stuck at act 2, chapter 3 - 'unbreakable' tried to restart, different versions, canary and master, no success ![Capture](https://user-images.githubusercontent.com/37451707/256010098-e5c08947-b4e8-47d9-8248-608607ce0786.PNG) [xenia.log](https://github.com/xenia-project/game-compatibility/files/12164813/xenia.log)

Getting an issue at the same act and chapter although the error is different.
![Xenia Error](https://github.com/xenia-project/game-compatibility/assets/121761304/59271b50-4f48-41ad-85b8-a58886457692)

**Comment by Potato302** [comment]
Got this error a chapter later
![Xenia Error](https://github.com/xenia-project/game-compatibility/assets/121761304/c2288c75-f926-4393-9fda-b932a7ef8371)

**Comment by Urielventris1984** [comment]
yep  having issues on this one  i have an amd pc, with amd 6800m  and when i play gears 3 it begins and crashes saying this : 
![image](https://github.com/xenia-project/game-compatibility/assets/142652741/9f03be63-77e4-4325-9579-38b0d70facff)

**Comment by dante3732** [comment]
> yep having issues on this one i have an amd pc, with amd 6800m and when i play gears 3 it begins and crashes saying this : ![image](https://user-images.githubusercontent.com/142652741/261823173-9f03be63-77e4-4325-9579-38b0d70facff.png)

Do not hope for a fix.

Maybe the vulkan will be finished in 5 years and it will be fine there.

**Comment by m013** [comment]
> stuck at act 2, chapter 3 - 'unbreakable' tried to restart, different versions, canary and master, no success ![Capture](https://user-images.githubusercontent.com/37451707/256010098-e5c08947-b4e8-47d9-8248-608607ce0786.PNG) [xenia.log](https://github.com/xenia-project/game-compatibility/files/12164813/xenia.log)

manged to fix this by trying another version of the game

**Comment by dharp117** [comment]
6950xt, 5800x3D Game keeps crashing. I expect this does not like AMD systems whatsoever.

**Comment by dante3732** [comment]
> 6950xt, 5800x3D Game keeps crashing. I expect this does not like AMD systems whatsoever.

Try xenia may 26 2020 or earlier and obviously disable rov.

**Comment by xXxTERRYxXx** [comment]
ACT 2 Trench Run seems to be broken af...
Right after you advance the trench section with the bombarder you reach a place whit some cells, in here depending on where you position the camera it will crash and close the emulator, on the first cinematic it crashes as well, you skip the cinematic then it will randomly crash and close while advancing the stage.
![act 2 trench run (crashes on first cinematic)](https://github.com/xenia-project/game-compatibility/assets/116123834/781b5c38-cff3-4e38-a304-f42454ca4db2)

**Comment by Voidwalker57** [comment]
- hello, when i start private multiplayer it works for a minute but crashes in the start. once when the bots and i loaded in ans second when i just started shooting.

**Comment by ankur20us** [comment]
> This is a known bug with Gears of War 3 kismet scripting in Xenia You have three options:
> 
> * Restart the chapter and play the entire way through without dying once.
> * Import a 100% savegame and start the game beyond the chapter that is bugged.
> * Using the Gears of War 2 Editor, edit the Coalesced_Int.bin file to unlock all chapters.

One of the most annoying bug, I restarted the chapter and cleared but unfortunately died so, thanks for the community I am uninstalling this idiot thing

**Comment by SloaJay** [comment]
![lights issue](https://github.com/xenia-project/game-compatibility/assets/85364852/46a323e2-d03b-4ab6-a6e3-9c77e4c1e5cf)
Lens flare issue is still present. Build is canary_experimental@5ca173cd4
Otherwise works ok, i'm thinking. Still playing through.

**Comment by JACsparr** [comment]
The game is now fully playable from start to finish on AMD GPUs. I am using an RX 6750XT, and while there are still some lighting issues and the game can crash (especially at the beginning and in Act 4 Part 6 "Tumba Aguada"), I recommend using Vulkan in these sections.
![Bug](https://github.com/xenia-project/game-compatibility/assets/171706123/4d583ee6-652e-43f0-a340-735a41d8bef7)
![bug 2](https://github.com/xenia-project/game-compatibility/assets/171706123/fa683558-509b-4408-95f2-ed990177fc83)

**Comment by Connor10121993** [comment]
New here, my interest in Xenia has been the Gears of War games running on steam deck using Xenia proton. It's been hard finding information so this may help others. From a YouTube video on Gears 2 where upon starting game on steam deck would result in a black screen with just the UI and gun fire I learned you needed to add a save from win 10 Xenia and replace the steam deck version. Tested this on Gear 3 and it also works.

GPU set to vulkan
Patch activated for unlocked fps and enabled to turn off most settings and turn on the dynamic resolution also set sync to off.

game ran between 20-30 fps slowdowns to low 20s during combat.
no combat ares hits mid 40fps
havent tried dx12 yet may get better performance.

Graphic glitches same as Gears 2 white flashing textures. But otherwise it was playable.

Going to test the same method using win 10 save on Gears Judgment next.

**Comment by Dethcorps51** [comment]
> New here, my interest in Xenia has been the Gears of War games running on steam deck using Xenia proton. It's been hard finding information so this may help others. From a YouTube video on Gears 2 where upon starting game on steam deck would result in a black screen with just the UI and gun fire I learned you needed to add a save from win 10 Xenia and replace the steam deck version. Tested this on Gear 3 and it also works.
> 
> GPU set to vulkan Patch activated for unlocked fps and enabled to turn off most settings and turn on the dynamic resolution also set sync to off.
> 
> game ran between 20-30 fps slowdowns to low 20s during combat. no combat ares hits mid 40fps havent tried dx12 yet may get better performance.
> 
> Graphic glitches same as Gears 2 white flashing textures. But otherwise it was playable.
> 
> Going to test the same method using win 10 save on Gears Judgment next.

Hi, also new. My interest is the same. I got gears 1 running just fine no issues. Now I have the same problem with 2, 3 and judgment. I get into the first mission of the campaign and it just a black screen with ui and gunfire when I press the fire button. Could you show the file structure for where the save file goes?

**Comment by Connor10121993** [comment]
> > New here, my interest in Xenia has been the Gears of War games running on steam deck using Xenia proton. It's been hard finding information so this may help others. From a YouTube video on Gears 2 where upon starting game on steam deck would result in a black screen with just the UI and gun fire I learned you needed to add a save from win 10 Xenia and replace the steam deck version. Tested this on Gear 3 and it also works.
> > GPU set to vulkan Patch activated for unlocked fps and enabled to turn off most settings and turn on the dynamic resolution also set sync to off.
> > game ran between 20-30 fps slowdowns to low 20s during combat. no combat ares hits mid 40fps havent tried dx12 yet may get better performance.
> > Graphic glitches same as Gears 2 white flashing textures. But otherwise it was playable.
> > Going to test the same method using win 10 save on Gears Judgment next.
> 
> Hi, also new. My interest is the same. I got gears 1 running just fine no issues. Now I have the same problem with 2, 3 and judgment. I get into the first mission of the campaign and it just a black screen with ui and gunfire when I press the fire button. Could you show the file structure for where the save file goes?

Hello,
So i used emudeck to install Xenia. The save file locations for me are in the emulation folder emu deck installs. Emulation/saves/xenia/saves 
There you will find folder the numbers correspond to saves you generate on windows just overwrite with windows files you made. 

I made the saves on windows using xenia canary and the save location was generated in the folder I put xenia.

if you didnt install using emudeck then should be a saves folder where you manually put xenia

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/39
